internal/fluentbit/config/builder/input.go

```diff
@@ -28,16 +28,18 @@ func createInputSection(pipeline *telemetryv1alpha1.LogPipeline, includePath, ex
 func createIncludePath(pipeline *telemetryv1alpha1.LogPipeline) string {
 	var toInclude []string
```

Comment:
It took me quite some time to understand the meaning of the variables `toInclude` and `toExclude`. I know it's old code, but since we are touching it, maybe let's change them to `includePath`/`excludePath`?

Reply:
Makes sense. Renamed it